package.json

```diff
@@ -65,6 +65,19 @@
         "metals.sbtScript": {
           "type": "string",
           "markdownDescription": "Optional absolute path to an `sbt` executable to use for running `sbt bloopInstall`.\n\nBy default, Metals uses `java -jar sbt-launch.jar` with an embedded launcher while respecting `.jvmopts` and `.sbtopts`.\n\nUpdate this setting if your `sbt` script requires more customizations like using environment variables."
+        },
+        "metals.scalafmtConfigPath": {
+          "type": "string",
+          "markdownDescription": "Optional custom path to the .scalafmt.conf file.\n\nShould be relative to the workspace root directory and use forward slashes `/` for file separators (even on Windows)."
+        },
+        "metals.compileOnSave": {
```

Comment:
We can leave this out for now, I will pick up cascade compilation again after the holidays.